libavfilter/dnn/dnn_backend_common.h

```diff
@@ -26,6 +26,26 @@
 
 #include "../dnn_interface.h"
 
+// one task for one function call from dnn interface
+typedef struct TaskItem {
+    void *model;
```

Comment:
add a simple comment to explain what model is

Reply:
Sure